graph/src/runtime/mod.rs

```diff
@@ -243,70 +245,94 @@ pub enum IndexForAscTypeId {
     NearChunkHeader = 84,
     NearBlock = 85,
     NearReceiptWithOutcome = 86,
-
-    // Tendermint Type IDs
+    // Reserved discriminant space for more Near type IDs: [87, 999]:
+    // Continue to add more Near type IDs here.
+    // e.g.:
+    // NextNearType = 87,
+    // AnotherNearType = 88,
+    // ...
+    // LastNearType = 999,
+
+    // Reserved discriminant space for more Ethereum type IDs: [1000, 1499]
+    TransactionReceipt = 1000,
+    Log = 1001,
+    ArrayH256 = 1002,
+    ArrayLog = 1003,
```

Comment:
I assume these are all not in real use yet?

Reply:
Positive.
Those types were created for #3373, which haven't gone live yet.